### DISCUSSION TITLE
Specifying a table name when using `--tables` and `--wheres` in the CLI

### DISCUSSION THREAD

**Discussion by nicholaslopez754** [planetscale/discussion#639] [FAQs] [answered]
Hi, I was wondering if it was possible to specify a table name when using `--wheres` and `--tables`. The use case here would be to do a database dump of specific tables and applying a few filters on a couple of tables.

I tried something to the effect of: 
```
pscale database dump  ... --tables table_1,table_2  --wheres "table_2.col_name is null"
``` 
```
pscale database dump  ... --tables table_1,table_2  --wheres "col_name is null"
``` 

but both error when it attempts to apply the `where` on `table_1`, but this works as expected if I only asked for `--tables table_2`

**Comment by jgreet** [comment, marked as answer]
Hi, try entering one where statement per table, as a comma separated list, in the same order as the provided table names, 

For example:
```sh
pscale db dump beam main --tables Account,Post --wheres "id = 'cl29ecsgb0014bvp9yxnazqbz',title = 'another one'"

Starting to dump tables 'Account,Post' from database beam to folder /Users/jgreet/tmp/pscale_dump_beam_main_20240627_145850
Dumping is finished! (elapsed time: 1.103153583s)
```

**Reply by nicholaslopez754** [reply]
Heya! Thanks for the response. In your example, what if I still wanted to dump the full table of accounts, but a filtered set of posts?

**Reply by jgreet** [reply]
You could try a where clause that always evaluates to true like `where true`:

```sh
pscale db dump beam main --tables Account,Post --wheres "true,title = 'another one'"
```

**Reply by nicholaslopez754** [reply]
Gotcha, thank you!
